Submitted By:            DJ Lucas <dj AT linuxfromscratch DOT org>
Date:                    2004-11-27
Initial Package Version: 1.4.2
Origin:                  Sent by Hamish Greig (Formerly of SourceMage)
                         Rediffed for 1.5.0.11 by Randy McMurchy
Description:             Prevents building of the debug image, saving approximately
                         200 MB of disk space.


diff -Naur jdk-1.5.0.11-orig/j2se/make/common/Release.gmk jdk-1.5.0.11/j2se/make/common/Release.gmk
--- jdk-1.5.0.11-orig/j2se/make/common/Release.gmk	2007-03-19 03:59:26.000000000 +0000
+++ jdk-1.5.0.11/j2se/make/common/Release.gmk	2007-03-19 04:43:35.000000000 +0000
@@ -186,7 +186,7 @@
 compare-image-jre compare-image-jdk ::
 	@$(ECHO) ">>>Making "$@" @ `$(DATE)` ..."; 
 
-images:: sanity-images post-sanity-images image-jre image-jdk image-jdk-debug
+images:: sanity-images post-sanity-images image-jre image-jdk
 
 #
 # Sources we ship in the SDK.
